core/embed/rust/src/ui/model_tr/layout.rs

```diff
@@ -1567,9 +1565,16 @@ extern "C" fn new_show_homescreen(n_args: usize, args: *const Obj, kwargs: *mut
             kwargs.get(Qstr::MP_QSTR_notification)?.try_into_option()?;
         let notification_level: u8 = kwargs.get_or(Qstr::MP_QSTR_notification_level, 0)?;
         let skip_first_paint: bool = kwargs.get(Qstr::MP_QSTR_skip_first_paint)?.try_into()?;
+        let hold: bool = kwargs.get(Qstr::MP_QSTR_hold)?.try_into()?;
 
         let notification = notification.map(|w| (w, notification_level));
-        let obj = LayoutObj::new(Homescreen::new(label, notification))?;
+        let obj = LayoutObj::new(Homescreen::new(
+            label,
+            notification,
+            "".into(),
+            "Locking the device...".into(),
```

Comment:
Let's remove `loader_title` since it's empty? Perhaps we can also merge `loader_description` and `can_lock_device` into an `Option<T>`?

Reply:
Removed title in [69c9bbe](https://github.com/trezor/trezor-firmware/pull/3440/commits/69c9bbe15d4274558e78b39dbde2fb1460857bd5)

Second simplification done in [cfb80ea](https://github.com/trezor/trezor-firmware/pull/3440/commits/cfb80ea89106b7ea0b8fef330e70630ca505e57b)